help/en/html/doc/Technical/Patterns.shtml

```diff
@@ -33,6 +33,38 @@ structure and patterns by looking at older code pieces.
 This page attempts to describe the recommended structure and patterns,
 and point to examples of current best practices.
 
+<a id="namedbeans" name="namedbeans"></a><h2>Names, NamedBeans, and Managers</h2>
```

Comment:
Would be interest in phrasing that would make clear whatever the issues were.

These really were JavaBeans when they were first created 17 years ago. At that point, the definition was mostly about
- has set/get methods for parameters, along with standard listener support for (any) bound parameters
- Serializable

Early JMRI worked with the JavaBean edit capabilities in NetBeans quite nicely. 

But we pretty rapidly outran the Serializable capabilities, esp. in our cross-platform, low-tech-strength user environment.  The solution to that was to create the ConfigureXML persistence system, and allow Serializable to atrophy. 

Later, some new NamedBeans come along (Logix, then others in that code family) that aren't as well-contained and as well defined. They're a long way from JavaBeans, but to be fair perhaps they shouldn't be NamedBeans either; simple signal logic isn't. 

It would be good if some somebody added text about what restrictions/issues were encountered. 

Are those big enough to move back toward having JavaBeans?